### PR TITLE
Add NodeJS and NPM to agent

### PR DESCRIPTION
**Pull request by gnadaban**

### CODE REVIEW

**Comment by gnadaban** [comment]
Hey @blueskyjunkie , before you merge, can you please make sure to configure the repo environment variables for the CI?

- DOCKERHUB_USERNAME
- DOCKERHUB_TOKEN

File: Dockerfile [hunk 1]
```diff
@@ -19,6 +19,8 @@ RUN apt-get update \
         libunwind8 \
         lsb-release \
         netcat \
+        nodejs \
```
Comment (blueskyjunkie) [inline Dockerfile]: note that it is Dockerfile best practice to pin package versions as well

**Review by blueskyjunkie — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (gnadaban) [reply Dockerfile]: Oh, ok. All of them?

Comment (gnadaban) [reply Dockerfile]: I've pinned NodeJS and NPM for now.

**Pull request merged**